### PR TITLE
feat(slice-machine): add GitHub settings page

### PR DESCRIPTION
**Pull request by bapmrl**
Completes DT-1804 and DT-1848.

### CODE REVIEW

File: packages/manager/src/index.ts [hunk 1]
```diff
@@ -13,6 +13,8 @@ export { createSliceMachineManager } from "./managers/createSliceMachineManager"
 export { createSliceMachineManagerMiddleware } from "./managers/createSliceMachineManagerMiddleware";
 export type { CreateSliceMachineManagerMiddlewareArgs } from "./managers/createSliceMachineManagerMiddleware";
 
+export type { GitRepoSpecifier, Owner } from "./managers/git/types";
```
Comment (angeloashmore) [inline packages/manager/src/index.ts]: Could we rename `Owner` to `GitOwner`? A top-level export named `Owner` has ambiguous meaning.

Comment (angeloashmore) [reply packages/manager/src/index.ts]: The rename can happen throughout the code, not just as an export rename.

Comment (bapmrl) [reply packages/manager/src/index.ts]: Okay, I'll do it ^^

**Comment by bapmrl** [comment]
Closed in favour of https://github.com/prismicio/slice-machine/pull/1285.

**Pull request closed without merging**